test/stat_tracker_test.rb

```diff
@@ -209,4 +232,23 @@ def test_it_can_find_the_worst_offensive_team
   def test_it_can_get_lowest_scoring_home_team
     assert_equal "Sporting Kansas City", @@stat_tracker.lowest_scoring_home_team
   end
+
+  # ==================       Team Stat Methods Tests     ==================
+
+  def test_it_gets_team_info
+    expected = {
+                id: 3,
+                franchise_id: 10,
+                name: "Houston Dynamo",
+                abbreviation: "HOU",
+                link: "/api/v1/teams/3"
+                }
+
```

Comment:
@guardiac not sure if this matters, but the spec asks for `team_id` and `team_name` instead of just `id` and `name`.